src/main/java/com/artgallery/ArtService.java

```diff
@@ -50,4 +57,39 @@ public void buyPainting() {
         painting.setAuthor(author);
         repository.addPainting(painting);
     }
+
+    public void sellPainting() {
+        System.out.println("Selling painting...");
+        List<Painting> paintings = repository.getPaintings();
+        showMyCollectionShort();
```

Comment:
to powinno wyswietlac na podstawie przekazanej listy showMyCollectionShort(paintings);